### PR TITLE
Add details to docs

### PR DESCRIPTION
**Pull request by pepopowitz**
1. (and most important) Adds more details about the workflows especially.
2. Attempts to unify hierarchy & language of section titles, based on how the docs made sense to me. If I did a bad job with this part, let me know and I'll just remove all that stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,31 +47,50 @@ $ pear known
...
 
 Now that we have a known author we can add them to what we're working on:
 
 ```
 $ pear current:add jonallured
+```
+
+This sets us up to give credit in our commits to `jonallured`. 
+
+Note: `pear` supports multiple authors, so you can add as many authors to your current pairing session as you want!
```
Comment (pepopowitz) [inline README.md:60]:
```suggestion
Note: Pear supports multiple authors, so you can add as many authors to your current pairing session as you want!
```

Looks like I did a bad job of recognizing the branding

File: README.md [hunk 2]
```diff
@@ -47,31 +47,50 @@ $ pear known
...
+```
 $ pear current:trailer
 Co-authored-by: Jonathan Allured <jon.allured@gmail.com>
 ```
 
-So yeah, now we can produce a trailer that will be picked up by GitHub and mark
-the commit as being authored by two developers. :sunglasses:
+Manually copy/paste that trailer into your commit message, and your pair(s) will get credit!
+
+### 2. Amend the current commit with `pear`
```
Comment (pepopowitz) [inline README.md:77]:
```suggestion
### 2. Amend the current commit with Pear
```

**Review by pepopowitz — COMMENTED** [review]
Oops, I missed that we spell it Pear.

**Comment by jonallured** [comment]
Ty!! I looked at the README again and I was not consistent in how I referred to the project - some "pear", some "Pear" so I'll merge this and then fix that. 👍

**Pull request merged**